CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # PD Change Log
 
 ## v2.0.1
+### New Feature
+* Add `Scatter Range` scheduler to balance the regions with specify key range
```

Comment:
-> Add the `Scatter Range` scheduler to balance Regions with the specified key range